Makefiles: Remove unused variables/targets

MoonlightSentinel opened the pull request:
Removes multiple unused elements that were declared but never used (refer to the commit messages for details concerning deleted rules)

MoonlightSentinel committed:
Makefiles: Remove unused OBJ_MSVC variable

These additional objects are already handled in build.d.

MoonlightSentinel committed:
Makefiles: Remove glue.lib

The library would contain no objects since GLUEOBJ is empty.

MoonlightSentinel committed:
Makefiles: Remove unused LFLAGS variable

This variable was set but never used

MoonlightSentinel committed:
posix.mak: Remove empty ROOTSRCC variable

MoonlightSentinel committed:
posix.mak/win32.mak: Remove unused variables

marler8997 reviewed (approved):
Thanks for the help.

MoonlightSentinel committed:
win32.mak: Remove unused pvs target

It's a relict from the days when DMD was implemented in C(++) and has been
broken for a long time now (it refers to cod3.c which was ported to D over
14 months ago)
